admin/spec/service/data-access/azure-file.data.service.spec.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
```

Comment:
what are you testing in this file?

Reply:
In case azure storage connection string is defined in the environment the tests will check the response objects for both download and upload methods.
In the opposite scenario they will check the mock response they receive.

I am questioning though whether these tests are really useful as they expect a container that perhaps would be better to provide through config.

@GuyHarwood thoughts?

Reply:
You are testing a mock when the connection string is not present.
When the connection string is present, you are testing a library maintained by a 3rd party